src/app/myTraining/workout/createWorkout/CreateWorkoutPage.tsx

```diff
@@ -11,7 +11,7 @@ const CreateWorkoutPage = () => {
   const initialWorkout: CreateWorkout = {
     name: routineData.routine.name,
     routineId: routineData.routine.id,
-    date: dayjs(),
+    date: dayjs().toDate(),
```

Comment:
dont change it yet, its not the problem about it. You will probably break the app

Reply:
I mean it didnt break the app, but if you say so I will delete this change